LocalSettings.php

```diff
@@ -3224,6 +3224,18 @@
 	'wgMinervaAlwaysShowLanguageButton' => [
 		'default' => true,
 	],
+        'wgMinervaTalkAtTop' => [
```

Comment:
Indentation

Reply:
Okay, I’m confused, do I need to add more indentation, or remove it?

Reply:
Well, you have alot of spaces this should be tabbed not spaces.

Reply:
Ohhh, I get it now. Alright, I’ll put tabs.

Reply:
Done.

Reply:
Indentation is still messed up, it doesn't use spaces anymore but still incorrectly indented.

Reply:
How about now?

Reply:
That's not it either.

Reply:
Still not it.

Reply:
Sorry if I’m wasting your time.

Reply:
Is it good now?

Reply:
No